### ISSUE TITLE
[Odin] Image Preview on File Chooser

### ISSUE THREAD

**Issue by phenobarbital** [elementary/files#2008]
### What Happened?

I'm upgraded my desktop (a fresh installation) from Hera to Odin and I lost the functionality to see image previews on File Chooser.
trying to attach an image file using the file chooser, is missing the image preview from previous versions.
Also, pantheon-photos crash when I try to open an image, but only happens in directories with many images.

### Steps to Reproduce

1.- install Odin and re-use the same /home folder from the previous version (Hera)
2.- Simply try to use the file chooser and click over an image

### Expected Behavior

In previous version (Hera) the Image Preview on File Chooser works.

### OS Version

6.x (Odin)

### Software Version

Latest release (I have run all updates)

### Log Output

```shell
The syslog when pantheon-photos crashed is:

Feb  8 17:45:42 prozac kernel: [17192.384023] io.elementary.p[53667]: segfault at 7e0 ip 00007ff97ad0f4b5 sp 00007fff255e3b90 error 6 in libc-2.31.so[7ff97acc9000+178000]
Feb  8 17:45:42 prozac kernel: [17192.384033] Code: 0f 85 29 2c 00 00 8b 8d 58 fa ff ff 83 f9 2f 0f 87 4d 17 00 00 89 ca 83 c1 08 48 03 95 68 fa ff ff 89 8d 58 fa ff ff 48 8b 12 <89> 02 e9 a8 c9 ff ff 48 89 e0 48 39 c4 74 15 48 81 ec 00 10 00 00
```

### Hardware Info

Latest Odin version.
Thinkpad P50 (i7 6th gen) with NVIDIA Quadro M2000M and official 510 drivers installed from apt.

**Comment by jeremypw** [comment]
The Odin FileChooser file chooser is a modified Gtk3 FileChooser so it would probably be possible to add an image preview widget to it but I understand that in Gtk4 the FileChooser will not support preview widgets so  unless it implements something natively I am not sure what can be done (apart from writing a completely new widget to replace the Gtk one).

**Comment by Gabriel-p** [comment]
So I guess this is a no go in Horus too? That's a shame, the preview is a really useful feature

**Comment by shiruken1** [comment]
Does #1601 close this one @jeremypw ?

**Comment by jeremypw** [comment]
@shiruken1 No, I don't think so - your PR applies only to the Files app not the FileChooser service.  However, if we get to the point of completely replacing the Gtk FileChooser with a Files one (see #2679) then we could expose your work there.
